articles/azure-databricks/quickstart-create-databricks-workspace-vnet-injection.md

```diff
@@ -14,7 +14,7 @@ ms.date: 12/04/2019
 
 The default deployment of Azure Databricks creates a new virtual network that is managed by Databricks. This quickstart shows how to create an Azure Databricks workspace in your own virtual network instead. You also create an Apache Spark cluster within that workspace. 
 
-For more information about why you might choose to create an Azure Databricks workspace in your own virtual network, see [Deploy Azure Databricks in your Azure Virtual Network (VNet Injection)] (/databricks/administration-guide/cloud-configurations/azure/vnet-inject).
+For more information about why you might choose to create an Azure Databricks workspace in your own virtual network, see [Deploy Azure Databricks in your Azure Virtual Network (VNet Injection)] (https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/vnet-inject).
```

Comment:
```suggestion
For more information about why you might choose to create an Azure Databricks workspace in your own virtual network, see [Deploy Azure Databricks in your Azure Virtual Network (VNet Injection)] (https://docs.microsoft.com/azure/databricks/administration-guide/cloud-configurations/azure/vnet-inject).
```

Reply:
Please remove locale.

Reply:
The locale has been removed by @ktoliver. Resolving this.